IMG/cpio/ventoy/hook/debian/pve-wa.sh

```
@@ -0,0 +1,20 @@
+
+LINE=$(grep -n 'proc.*proc.*proc' $script | awk -F':' '{print $1}')
+
+sed -n "1,${LINE}p" $script >> $tmpscript
+
+echo "if grep -q 'rdinit=/vtoy/vtoy' /proc/cmdline; then"    >> $tmpscript
+echo "    sed 's#rdinit=/vtoy/vtoy##g' /proc/cmdline > /etc/cmdline"    >> $tmpscript
+echo "    mount --bind /etc/cmdline  /proc/cmdline"    >> $tmpscript
+echo "fi"    >> $tmpscript
+
+
+let LINE++
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'let' is undefined.

(SC3039)